include/zephyr/net/wifi.h

```diff
@@ -49,6 +49,8 @@ enum wifi_security_type {
 	WIFI_SECURITY_TYPE_WEP,
 	/** WPA-PSK security. */
 	WIFI_SECURITY_TYPE_WPA_PSK,
+	/** WPA/WPA2/WPA3 PSK security. */
+	WIFI_SECURITY_TYPE_WPA_AUTO,
```

Comment:
WPA_AUTO_PERSONAL might limit to personal security.

Reply:
Done